src/components/Book/CategoryView.tsx

```diff
@@ -7,16 +9,28 @@ type Props = {
 };
 export function BookCategoryView({ book }: Props) {
     const bestClass = book.bestInCategory ? ' best' : '';
+    const isDesktop = useMediaQuery({ query: `(min-width: 680px )` });
+    const [active, setActive] = React.useState(isDesktop ? true : false);
 
-    const className = classNames({
+    const booksClassName = classNames({
         book: true,
         best: book.bestInCategory,
         'mb-3': true,
     });
 
+    const coversClassName = classNames(active ? 'book-cover' : 'book-detail');
+    const prefacesClassName = classNames(
+        'bg-body-tertiary',
+        active ? 'preface' : 'preface-detail'
+    );
```

Comment:
You can override class definitions for different breakpoints.

- Why did you used this way?
- What did you come across while using CSS breakpoints and it did not work?

Reply:
handled with css breakpoints along with toggler